components/Logo/Logo.jsx

```diff
@@ -5,11 +5,11 @@ export default () => {
     <div className="navbar-header">
       <a href="/" className="navbar-brand">
          <strong className="theme-blue">YASP</strong>
-         <a data-toggle="collapse" data-target=".navbar-collapse" className="navbar-toggle">
+         <div data-toggle="collapse" data-target=".navbar-collapse" className="navbar-toggle">
```

Comment:
So, this is actually supposed to look like 
![image](https://cloud.githubusercontent.com/assets/375232/15620616/186d56bc-2410-11e6-8f96-2a6227d83fba.png)

I realized (from the live site) that the collapse stuff is for mobile, for toggling the menu on narrow screens. It shouldn't be a descendent of the main navbar brand, as it's floated to the right side on narrow displays.